### ISSUE TITLE
Loose chapters marked as read

### ISSUE THREAD

**Issue by ThePromidius** [Kareadita/Kavita#1157]
TLDR:
-> Tachiyomi sends chapter 0.0 when there's no chapters read.
-> Kavita marks all loose chapters as read

**Describe the bug**
When there is nothing read (when you just tracked the manga for the first time) tachiyomi sends a request and sends 0.0
our encoding starts at 0.001 not at 0.0
Also 0.0 are the specials which we don't track

The problem is that when 0.0 is passed, kavita marks as read all the chapters that are in volume 0 (loose chapters)

**To Reproduce**
```2022-02-26 15:16:07.497 4283-13064/eu.kanade.tachiyomi.debug
 I/okhttp.OkHttpClient: --> POST http://192.168.0.135:5000/api/Reader/mark-chapter-until-as-read?seriesId=1&chapterNumber=0.0
```
This is marking volume 0 as read, hence marking loose chapters as read.
This request is sent when tracking a serie in tachiyomi. So unless there's read chapters already, current progress is 0.0

**Expected behavior**
To not mark loose chapters as read

**Screenshots**
N/A

**Desktop (please complete the following information):**
N/A

**Smartphone (please complete the following information):**
N/A

**Additional context**
